ember-modifier/src/-private/function-based/modifier.ts

```diff
@@ -133,6 +134,7 @@ export default function modifier(
   // type of `setModifierManager` today is `void`; we pretend it actually
   // returns an opaque `Modifier` type so that we can provide a result from this
   // type which is useful to TS-aware tooling (e.g. Glint).
+  fn.toString = () => name || fn.name;
```

Comment:
@patricklx I guess this line should go above the comment `SAFETY: the cast here is ...` to avoid confusion